config-example.yaml

```diff
@@ -25,3 +25,9 @@ instanceDetails:
 videoSettings:
   # Change this value and keep it secure.  Treat it like a password to your live stream.
   streamingKey: abc123
+
+  # Enable YP to be listed in the Owncast directory and let people discover your instance.
+yp:
+  enabled: true
```

Comment:
Given my workflow of starting a new instance with `cp config-example.yaml config.yaml`, this effectively defaults to true. That's probably OK, we should just be mindful of it.

Reply:
Good point!  Changed.